components/Layout/Layout.jsx

```diff
@@ -1,16 +1,22 @@
 import Head from "next/head";
-import { Container, Flex } from "@chakra-ui/react";
+import { Container, Flex, Text } from "@chakra-ui/react";
 import { productionUrl } from "../../utils/constants";
 
+const titleBarHeight = "80px";
+
 export default function Layout({ children }) {
+
   return (
     <>
       <Head>
         <title>Emissions Calculator</title>
         <meta name="description" content="Emissions calculator" />
         <link rel="icon" href={`${productionUrl}/favicon.ico`} />
       </Head>
-      <Flex height="100vh" alignItems="center">
+      <Flex alignItems="center" minHeight={`calc(100vh - ${titleBarHeight})`} pos="relative" top={titleBarHeight} py={5}>
+        <Flex alignItems="center" height={titleBarHeight} pos="fixed" top="0" w="100%" bg="#055E9E" color="white" zIndex={10}>
+          <Text pl="3.5rem" fontSize="2.25rem" fontWeight="bold">SeeChange</Text>
+        </Flex>
```

Comment:
I think maybe all I would want changed is that the fixed position element be placed at the top of the body code, and outside of the 'content container'. Would work better visually for me. 
It would make me a touch nervous that the whole body wasn't wrapped up, but maybe this is a new era and I shouldn't worry about that so much..

Reply:
Sorry, I think I might be a little confused. 
Apart from moving the header out of the content container, did you mean you'd then prefer both those components wrapped in an additional overall container?

Reply:
I was thinking that an additional container might be good, but maybe that is not so necessary these days. Dunno. Could just wrap in `<></>` to demarcate the visible zone. Then again I'm not sure that achieves much.